pkgs/by-name/vl/vlc-bittorrent/package.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, autoconf-archive
+, autoreconfHook
+, boost
+, fetchFromGitHub
+, libtorrent-rasterbar
+, libvlc
+, openssl
+, pkg-config
+, stdenv
+}:
+
+# VLC does not know where the vlc-bittorrent package is installed.
+# make sure to have something like:
+#   environment.variables.VLC_PLUGIN_PATH = "${pkgs.vlc-bittorrent}";
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vlc-bittorrent";
+  version = "2.15.0";
+
+  src = fetchFromGitHub {
+    owner = "johang";
+    repo = "vlc-bittorrent";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-7FHeQYHbMKZJ3yeHqxTTAUwghTje+gEX8gSEJzfG5sQ=";
+  };
+
+  nativeBuildInputs = [
+    autoconf-archive
+    autoreconfHook
+    pkg-config
+  ];
+
+  buildInputs = [
+    boost
+    libtorrent-rasterbar
+    libvlc
+    openssl
+  ];
+
+  strictDeps = true;
+
+  postFixup = ''
+    # It's a library, should not have a desktop file
```

```suggestion
  # It's a library, should not have a desktop file
  postFixup = ''
```

This is a bit convoluted, but that's it:

The string inside `postFixup` is passed as a shell script to the inferior Bash process. It includes the comment.

If, by any reason, you need to update or reword the comment, the shell script checksum will change despite it being conceptually identical. 
And that change will trigger a rebuild.

So, it's better to put the comments outside the script.

---

Oh, thanks for pointing that out. Makes a lot of sense. Will fix tomorrow.